### ISSUE TITLE
How to get rid off the frame modifier - get the size of the button view

### ISSUE THREAD

**Issue by dirkfabisch** [costachung/neumorphic#3]
Again I try to get the smallest size of a button for dynamic text sizes. Could you point me in the right direction? I have no idea how to get the button size determined by the SF Symbol size.

```swift

struct HeaderButtonRowView: View {
    var colorScheme: ColorScheme

    var body: some View {
        NeoColors.shared.colorScheme = colorScheme
        return HStack {
            Button(action: {
                logger.info("cancel")
            }) {
                HStack(alignment: .center, spacing: 10) {
                    Image(systemName: "xmark")
                        .font(Font.title.weight(.bold))
                }
                
            }
            .softButtonStyle(RoundedRectangle(cornerRadius: 15, style: .continuous), mainColor: NeoColors.shared.background(), textColor: NeoColors.shared.accent(), darkShadowColor: NeoColors.shared.dark(), lightShadowColor: NeoColors.shared.light())
//            .frame(width: 50, height: 50)
            Spacer()
            Button(action: {
                logger.info("OK")
            }) {
                Image(systemName: "checkmark").font(Font.title.weight(.bold))
            }
            .softButtonStyle(RoundedRectangle(cornerRadius: 15, style: .continuous), mainColor: NeoColors.shared.background(), textColor: NeoColors.shared.accent(), darkShadowColor: NeoColors.shared.dark(), lightShadowColor: NeoColors.shared.light())
            .frame(width: 50, height: 50)

        }
        .padding()
    }
}

```

<img width="471" alt="Screen Shot 2020-05-31 at 16 52 57" src="https://user-images.githubusercontent.com/5524851/83355341-36830c00-a35f-11ea-96b4-71e8536b0e18.png">

**Comment by costachung** [comment]
I think you are doing it right. 

The softButtonStyle doesn't work correctly without the frame modifier. Thanks for pointing this out. I will update the code :)

**Comment by costachung** [comment]
I've released a new version 1.1.0 that should fix the incorrect size issue. Please update and let me know if it is ok. Thank you @dirkfabisch !

**Comment by dirkfabisch** [comment]
Perfect!!! This new version fixed the issue.
